.vscode/cspell.json

```diff
@@ -37,49 +46,373 @@
+    "sdk/spring/spring-messaging-azure-servicebus/**",
+    "sdk/spring/spring-messaging-azure-eventhubs/**",
+    "sdk/storage/azure-storage-blob-batch/**",
+    "sdk/storage/azure-storage-blob-changefeed/**",
+    "sdk/storage/azure-storage-perf/**",
+    "sdk/storage/azure-storage-internal-avro/**",
+    "sdk/storage/azure-storage-common/**",
+    "sdk/storage/azure-storage-blob-cryptography/**",
+    "sdk/storage/azure-storage-queue/**",
+    "sdk/synapse/azure-analytics-synapse-accesscontrol/**",
+    "sdk/synapse/azure-analytics-synapse-managedprivateendpoints/**",
+    "sdk/synapse/azure-analytics-synapse-monitoring/**",
+    "sdk/synapse/azure-analytics-synapse-spark/**",
+    "sdk/storage/azure-storage-blob-nio/**",
+    "sdk/tables/.idea/**",
+    "sdk/template/azure-sdk-template/**",
+    "sdk/template/azure-sdk-template-three/**",
+    "sdk/template/azure-template-perf/**",
+    "sdk/template/azure-sdk-template-two/**",
+    "sdk/tables/azure-data-tables-perf/**",
+    "sdk/textanalytics/azure-ai-textanalytics-perf/**",
+    "sdk/tools/azure-sdk-archetype/**",
+    "sdk/tools/azure-sdk-build-tool/**",
+    "sdk/storage/azure-storage-file-share/**",
+    "sdk/translation/azure-ai-documenttranslator/**",
+    "sdk/videoanalyzer/azure-media-videoanalyzer-edge/**",
+    "sdk/synapse/azure-analytics-synapse-artifacts/**",
+    "sdk/webpubsub/azure-messaging-webpubsub/**",
+    "sdk/tables/azure-data-tables/**",
+    "sdk/textanalytics/azure-ai-textanalytics/**",
+    "sdk/storage/azure-storage-file-datalake/**",
+    "sdk/storage/azure-storage-blob/**",
+    "sdk/eventhubs/azure-messaging-eventhubs-stress/**",
+    "sdk/jdbc/azure-identity-providers-jdbc-mysql/**",
+    "sdk/jdbc/azure-identity-providers-jdbc-postgresql/**",
+    "sdk/maps/azure-maps-elevation/**",
+    "sdk/maps/azure-maps-geolocation/**",
+    "sdk/maps/azure-maps-timezone/**",
+    "sdk/search/.idea/**",
+    "sdk/spring/spring-cloud-azure-starter-jdbc-postgresql/**",
+    "sdk/spring/spring-cloud-azure-starter-jdbc-mysql/**",
+    "sdk/devcenter/azure-developer-devcenter/**"
   ],
   "words": [
+    "adal",
+    "amqp",
+    "AUHours",
+    "autoscale",
+    "autodetection",
     "azsdk",
-    "codesnippet",
-    "revapi",
+    "azsynapse",
+    "azurestackhci",
+    "backoff",
+    "boringssl",
+    "branchcoverage",
+    "cacerts",
     "checkstyle",
-    "spotbugs",
-    "OLTP",
+    "Classpath",
+    "cname",
+    "codesnippet",
+    "cols",
+    "contoso",
+    "contosomedia",
+    "createorupdate",
+    "creds",
+    "credscan",
+    "databind",
+    "databricks",
+    "ddos",
+    "deallocate",
+    "decryptor",
+    "Deserialization",
+    "destfile",
+    "Dgenerate",
+    "doclint",
+    "doclinting",
+    "doctitle",
+    "dlocal",
+    "dummyrg",
+    "eastus",
+    "embedme",
+    "encryptor",
+    "etag",
+    "eventhub",
+    "eventhubs",
+    "filereports",
+    "fileset",
+    "fout",
+    "fqdns",
+    "gdnbaselines",
```

Comment:
This word only shows up in the `eng/` folder according to my search. It should just be excluded.